lua/lspconfig/cssls.lua

```diff
@@ -27,6 +27,19 @@ https://github.com/vscode-langservers/vscode-css-languageserver-bin
 ```sh
 npm install -g vscode-css-languageserver-bin
 ```
+
+Neovim does not currently include built-in snippets. `vscode-html-languageserver` only provides completions when snippet support is enabled. 
```

Comment:
Still need to fix this type :) vscode-html-languageserver -> vscode-css-languageserver